From ab9f289d925262c369c6611f0ecaf1f63c6a738c Mon Sep 17 00:00:00 2001
From: Debarshi Ray <debarshir@gnome.org>
Date: Wed, 1 Jul 2015 16:45:12 +0200
Subject: [PATCH] widget: Don't hide the mouse pointer

Although the code has been there for a while, users have recently
reported that the pointer remains hidden even after moving the mouse.
This is more irritating than having the pointer remain visible while
using the terminal, so let's disable this code until a fix shows up.

https://bugzilla.gnome.org/show_bug.cgi?id=725342
---
 src/vte.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/vte.c b/src/vte.c
index 906d799..7ea3a8b 100644
--- a/src/vte.c
+++ b/src/vte.c
@@ -2189,6 +2189,7 @@ _vte_terminal_set_pointer_visible(VteTerminal *terminal, gboolean visible)
 	GdkWindow *window;
 	struct vte_match_regex *regex = NULL;
 
+	visible = TRUE;
 	terminal->pvt->mouse_cursor_visible = visible;
 
         if (! gtk_widget_get_realized (&terminal->widget))
-- 
2.1.0

